app/scripts/services/logout.js

```diff
@@ -0,0 +1,32 @@
+angular
+  .module('confRegistrationWebApp')
+  .service('logoutService', function(
+    $location,
+    $cookies,
+    $window,
+    $rootScope,
+    $sce,
+  ) {
+    this.logoutFormProviders = function(response) {
+      switch ($cookies.get('crsAuthProviderType')) {
+        case 'FACEBOOK':
+          // if facebook, then logout from facebook using url generated on the backend
+          if (response.data) {
+            $window.location.href = response.data.url;
+          }
+          break;
+        case 'RELAY':
+          // if relay, then then redirect to the Relay logout URL
+          var serviceUrl = $location.absUrl().replace('logout', '');
```

Comment:
could probably be a const instead of var.

Reply:
fixed